packages/studio/src/store/hotReloadStore.ts

```diff
@@ -81,6 +82,15 @@ function syncPages(studioData: StudioData) {
   });
 }
 
+function syncLayouts(studioData: StudioData) {
+  const layoutNameToLayoutState = removeTopLevelFragments(
```

Comment:
We added back in the generic param to `removeTopLevelFragments`. We figured we might need it for Layouts. I think we want to change the signature of `removeTopLevelFragments` from `removeTopLevelFragments<T extends PageState>` to `removeTopLevelFragments<T extends PageState | LayoutState>`.

Reply:
Updated!